README.md

```diff
@@ -72,26 +72,32 @@ Once [installed](#install), you can use the following code to process an example
 user lists by sending a (RESTful) HTTP API request for each user record:
 
 ```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
 $browser = new React\Http\Browser();
 
 $concurrency = isset($argv[1]) ? $argv[1] : 3;
 
 // each job should use the browser to GET a certain URL
 // limit number of concurrent jobs here
-$transformer = new Transformer($concurrency, function ($user) use ($browser) {
+$transformer = new Clue\React\Flux\Transformer($concurrency, function ($user) use ($browser) {
     // skip users that do not have an IP address listed
     if (!isset($user['ip'])) {
         return React\Promise\resolve($user);
     }
 
     // look up country for this IP
     return $browser->get("https://ipapi.co/$user[ip]/country_name/")->then(
-        function (ResponseInterface $response) use ($user) {
+        function (Psr\Http\Message\ResponseInterface $response) use ($user) {
             // response successfully received
             // add country to user array and return updated user
             $user['country'] = (string)$response->getBody();
 
             return $user;
+        }, function (Exception $e) {
+            echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Comment:
Looks reasonable, but unfortunately changes the outcome of the promise as it may no longer reject, but would instead resolve with an `array` from the fulfillment handler or an implicit `null` from the rejection handler. (See also https://github.com/clue/reactphp-mq/pull/47#discussion_r1517879744)

I would rather remove this here and instead rely on the stream `error` handler near the bottom instead. Similarly, we may want to update the error handlers in the other examples. WDYT?

```suggestion
```

Reply:
Thank you @clue for bringing this up. Please let me know, if everything works fine with you. 👍